keyboards/dumbpad/v3x/config.h

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2022 imchipwood && deveth0
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER      0x0010
+
```

Comment:
We're in the process of moving USB Device Descriptors to `info.json`.

```suggestion
```

Reply:
commit: https://github.com/qmk/qmk_firmware/pull/18663/commits/2086c250fab2950ce747b724ffd78b7e23d44a36